src/drivers/azure-storage-blob.ts

```diff
@@ -100,13 +129,40 @@ export default defineDriver((opts: AzureStorageBlobOptions) => {
         return null;
       }
     },
+    async getItemRaw(key) {
+      try {
+        const blob = await getContainerClient()
+          .getBlockBlobClient(key)
+          .download();
+        if (isBrowser) {
+          return blob.blobBody ? await blobToString(await blob.blobBody) : null;
+        }
+        return blob.readableStreamBody
+          ? await streamToBuffer(blob.readableStreamBody)
+          : null;
+      } catch {
+        return null;
+      }
+    },
     async setItem(key, value) {
       await getContainerClient()
         .getBlockBlobClient(key)
         .upload(value, Buffer.byteLength(value));
     },
+    async setItemRaw(key, value) {
+      await getContainerClient()
+        .getBlockBlobClient(key)
+        .upload(value, Buffer.byteLength(value));
+    },
     async removeItem(key) {
-      await getContainerClient().getBlockBlobClient(key).delete();
+      const exists = await getContainerClient()
```

Comment:
you can actually call `deleteIfExists({ deleteSnapshots: "include" })` iirc

Reply:
Great suggestion. Done